### PR TITLE
Query management feature

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8109

### Change description ###
Query management feature

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -93,7 +93,7 @@
     "@angular/ssr": "^17.3.6",
     "@cucumber/cucumber": "^10.8.0",
     "@edium/fsm": "^2.1.2",
-    "@hmcts/ccd-case-ui-toolkit": "7.0.57",
+    "@hmcts/ccd-case-ui-toolkit": "7.0.56-query-management",
```
Comment (andywilkinshmcts) [inline package.json:96]: You've gone backwards -ok for now but be careful when you create your final toolkit build

File: src/cases/containers/query-management-container/query-management-container.component.ts [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+      if (htmlElement) {
+        htmlElement.scrollIntoView({ behavior: 'smooth', block: 'center' });
+        htmlElement.focus();
+      }
+    }
+  }
+
+  private getQueryCreateContext(): QueryCreateContext {
+    switch (this.queryItemId) {
+      case '1':
```
Comment (andywilkinshmcts) [inline src/cases/containers/query-management-container/query-management-container.component.ts:215]: I hate this switch statement. At least use constants for these magic numbers

File: src/work-allocation/containers/available-tasks/available-tasks.component.spec.ts [hunk 3]
```diff
@@ -101,7 +101,8 @@ describe('AvailableTasksComponent', () => {
         { provide: AllocateRoleService, useValue: mockRoleService },
         { provide: Store, useValue: storeMock },
         { provide: RpxTranslationService, useFactory: rpxTranslationServiceStub }
-      ]
+      ],
+      schemas: [NO_ERRORS_SCHEMA]
```
Comment (andywilkinshmcts) [inline src/work-allocation/containers/available-tasks/available-tasks.component.spec.ts:105]: This is usually a hack. Before release we should figure out why this is needed, and remove it unless there is some very special circumstance that requires it.

File: src/work-allocation/containers/task-list/task-list.component.spec.ts [hunk 4]
```diff
@@ -106,7 +107,8 @@ describe('TaskListComponent', () => {
         { provide: LoadingService, useValue: mockLoadingService },
         { provide: FeatureToggleService, useValue: mockFeatureToggleService },
         { provide: SessionStorageService, useValue: mockSessionStorageService }
-      ]
+      ],
+      schemas: [NO_ERRORS_SCHEMA]
```
Comment (andywilkinshmcts) [inline src/work-allocation/containers/task-list/task-list.component.spec.ts:111]: Hack, as in previous comment. Should be removed before release

**Review by andywilkinshmcts — APPROVED** [review]

**Review by OgunyemiO — DISMISSED** [review]

**Review by MunishSharmaHMCTS — APPROVED** [review]

**Pull request merged**